packages/react-aria-components/test/Tree.test.tsx

```diff
@@ -187,20 +188,14 @@ describe('Tree', () => {
     expect(tree).toHaveAttribute('style', expect.stringContaining('width: 200px'));
   });
 
-  it('should have the base set of aria and data attributes', () => {
+  it('should have the base set of data attributes', () => {
     let {getByRole, getAllByRole} = render(<StaticTree treeProps={{defaultExpandedKeys: 'none'}} />);
     let tree = getByRole('treegrid');
-    expect(tree).toHaveAttribute('data-rac');
```

Comment:
did you mean to get rid of this? or is this checked somewhere else and i just missed it

Reply:
Ah thanks, this one I accidentally removed, but the aria ones are tested elsewhere